contracts/flashMint/interface/IVatDaiFlashLender.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
```

Comment:
This file is not needed if you reference IERC3156FlashBorrower instead of implementation